animations/spells/1st/vitality-lash.json

```diff
@@ -0,0 +1,29 @@
+{
+	"item:slug:vitality-lash": [
+		{
+			"trigger": "spell-cast",
+			"preset": "ranged",
+			"options": {},
+			"presets": [
+				{
+					"predicate": [
+						"jb2a:free"
+					],
+					"file": "jb2a.energy_strands.range.standard.purple.04",
```

Comment:
Why specifically 04 and not the entire lineup?

Reply:
I just think 4 looks better as an animation (more like a lash)

1, 2 - Too Squiggly
3 - Too straight